### DISCUSSION TITLE
md5 32 encryption

### DISCUSSION THREAD

**Discussion by k3rwin** [projectdiscovery/nuclei-templates#7418] [Q&A]
How to use the md5 32-bit encryption method, can it be added？

**Comment by MetzinAround** [comment]
Hi @k3rwin , I'm not sure what you mean by adding an encryption method for a nuclei template. Could you describe it some more please?

**Comment by Mzack9999** [comment]
@k3rwin just like any other helper function:
```yaml
{{md5('some data')}}
```

You can find more examples in the `nuclei-templates` repository (for [example](https://github.com/projectdiscovery/nuclei-templates/blob/1e0744f23bd41c2bc3253c879ad6e7b2bee8e56c/http/cves/2022/CVE-2022-4447.yaml#L35))

**Reply by k3rwin** [reply]
Thanks guys, I'm using this for the first time to write a template and just found out how to use it
